### PR TITLE
fixes and docs

### PR DESCRIPTION
**Pull request by vogievetsky**

### CODE REVIEW

File: docs/druid-0.9.1-lookup-migration.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+```yaml
+  - name: druid
+    type: druid
+    host: your.broker.host
+    version: 0.9.1-legacy-lookups
+```
+
+Make sure to restart Pivot.
+
+In this configuration all the Druid 0.9.1 features will be used except the new lookups
```
Comment (lorem--ipsum) [inline docs/druid-0.9.1-lookup-migration.md]: Some sentences have an end dot, some don't.

**Comment by lorem--ipsum** [comment]
🍗

File: src/common/models/app-settings/app-settings.ts [hunk 2]
```diff
@@ -125,8 +125,13 @@ export class AppSettings implements Instance<AppSettingsValue, AppSettingsJS> {
 
   public toClientSettings(): AppSettings {
     var value = this.valueOf();
+
     value.clusters = value.clusters.map((c) => c.toClientCluster());
-    value.dataSources = value.dataSources.map((ds) => ds.toClientDataSource());
+
+    value.dataSources = value.dataSources
+      .filter((ds) => ds.isQueryable())
```
Comment (lorem--ipsum) [inline src/common/models/app-settings/app-settings.ts]: Okay this shouldn't have passed the tests : running them locally shows one failing tests on app-settings, directly linked to this (data-source mocks don't have an executor and therefore aren't queryable, which messes up the app-settings tests)... How come Travis didn't notice anything ? Come on Travis, you're embarrassing yourself.

Comment (vogievetsky) [inline src/common/models/app-settings/app-settings.ts]: Wow strange. Boo travis!

**Pull request merged**